modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/storage/LastSnapshotStorage.scala

```diff
@@ -37,6 +37,7 @@ object LastSnapshotStorage {
 
       def set(snapshot: Hashed[S], state: SI): F[Unit] =
         snapshotR.modify {
+          case None => ((snapshot, state).some, Applicative[F].unit)
```

Comment:
There is a `setInitial` function that performs that task. It's done like that on purpose so that setting first snapshot is   explicit and deliberate.

Reply:
@TheMMaciek Yes, that's where I took that line from. I wasn't sure if it was necessary that we be explicit. I have to look further at the code but sometimes it seems like it's not easy to determine that.

Can we talk more about this task so I know how to go about fixing it?